tests/Integration/Behaviour/Features/Scenario/Cart/CartToOrder/cart_to_order.feature

```diff
@@ -28,10 +28,10 @@ Feature: Check cart to order data copy
     When I select address "address1" in my cart
     When I select carrier "carrier1" in my cart
     When I validate my cart using payment module fake
-    Then current cart order total for products should be 20.6 tax included
-    Then current cart order total for products should be 19.81 tax excluded
-    Then current cart order total discount should be 10.3 tax included
-    Then current cart order total discount should be 9.91 tax excluded
+    Then current cart order total for products should be 20.604480 tax included
```

Comment:
How come in some cases you have 6 digits and in other only 1 (or 2 or 3) Is it because the computation is exactly one digit or are there other rounding rules? If it's an exact value maybe add a pending 0 to avoid confusion?

Reply:
Since we increased the precision, in this case we can have up to 6 digits, but that doesn't mean we will always have 6 digits, the end result could only have one digit, I believe this is normal 🤔